### PR TITLE
14955 - remove all Sentry logging calls

### PR DESCRIPTION
**Pull request by jonathan-longe**
*Issue #:* /bcgov/entity14955

Remove all Sentry logging calls.  Add docker-compose.yml file to spin up two Postgres databases required for testing.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/docker-compose.yml [hunk 1]
```diff
@@ -4,10 +4,9 @@ services:
...
     environment:
-      - LC_ALL=C.UTF-8
-      - POSTGRES_DB=legal_test
-      - POSTGRES_USER=legal
-      - POSTGRES_PASSWORD=
-      - POSTGRES_PORT=5432
+      LC_ALL: "C.UTF-8"
+      POSTGRES_DB: "${DATABASE_TEST_NAME}"
+      POSTGRES_USER: "${DATABASE_TEST_USERNAME}"
+      POSTGRES_PASSWORD: "${DATABASE_TEST_PASSWORD}"
```
Comment (jonathan-longe) [inline legal-api/docker-compose.yml:12]: Using this opportunity to clean up our tooling.  This change is unrelated to the ticket.  I can remove the file if anyone objects to its inclusion.

File: queue_services/entity-emailer/.env.sample [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+# ## NATS - STAN - DEV
+#NATS_SERVERS=
+#NATS_CLIENT_NAME=
+#NATS_CLUSTER_ID=
+#NATS_EMAILER_SUBJECT=
+#NATS_ENTITY_EVENT_SUBJECT=
+#NATS_QUEUE=
+
+#DEPLOYMENT_ENV=
```
Comment (jonathan-longe) [inline queue_services/entity-emailer/.env.sample:84]: If an .env file is needed, then I suggest we include a sanitized copy so it's more obvious that an .env is required.

**Review by argush3 — APPROVED** [review]
looks good.  the only comment I really have is that we usually add the ticket number to the PR title and the git commit message.

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Review by vysakh-menon-aot — APPROVED** [review]

File: queue_services/entity-emailer/src/entity_emailer/config.py [hunk 3]
```diff
@@ -60,8 +60,6 @@ class _Config():  # pylint: disable=too-few-public-methods
 
     PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
-    # monitoring
-    SENTRY_DSN = os.getenv('SENTRY_DSN', None)
```
Comment (severinbeauvais) [inline queue_services/entity-emailer/src/entity_emailer/config.py:64]: Can we remove this key from the environment (once this change is deployed)?

Or perhaps just remove the sentry application from [here](https://github.com/bcgov/lear/blob/f2f7c2caa42973c9949a39d18a6309f24955fbeb/queue_services/entity-emailer/devops/vaults.json#L15)?

Ask Patrick Wei for confirmation that this is the right thing to do.

**Pull request merged**